argus/backend/service/client_service.py

```diff
@@ -24,7 +30,24 @@ def get_model(self, run_type: str) -> PluginModelBase:
 
     def submit_run(self, run_type: str, request_data: dict) -> str:
         model = self.get_model(run_type)
-        model.submit_run(request_data=request_data)
+        run = model.submit_run(request_data=request_data)
+        if run.assignee and run.test_id:
+            self.notification_manager.send_notification(
+                receiver=run.assignee,
+                sender=g.user.id,
```

Comment:
what is g.user.id in the scope of ArgusClient?

Reply:
It's the ClientService scope of argus application and `g` is the global flask context variable.

Anyway I'll remove this notification in particular, as now I think it's a bit too excessive to have a notification created on *every single* run submitted to argus.

Reply:
yes, might be too excessive and people will ignore it. Please remove unrelated imports then too.
I think would be good to send once a day a reminder about jobs to investigate.

Reply:
Done